### DISCUSSION TITLE
是否添加自动检测模组兼容性

### DISCUSSION THREAD

**Discussion by HomoobyDX** [Meloong-Git/PCL#619] [提问] [answered]
你们玩私包时会遇到什么启动器都启动不了，日志没法提供任何帮助的情况吧，模组兼容性导致的，但一个一个去百科找，去bili找很麻烦，正所谓pcl是一款很好的启动器，能为玩家服务是最好的选择
所以推荐添加自动检测游戏模组兼容性

**Comment by ZerkyLiu** [comment]
那PCL怎么检查模组兼容性？一些模组连curseforge上也没有，PCL只能提供一些建议，具体还是要看错误报告。（只是我个人的观点，不要全信！）

**Comment by ruattd** [comment, marked as answer]
PCL2 与你自己去找的方式是一样的，你找不到它也找不到...
Mod 文件中一般不会加入不兼容列表（除非作者闲得蛋疼下载一堆 Mod 挨个试），PCL2 唯一能做到的就是把那些自带不兼容列表或者百科网站上有不兼容列表的 Mod 找出来。
但这种 Mod 屈指可数，只有一些知名 Mod 会把不兼容的另一些知名 Mod 放进不兼容列表里，百科也只有一些知名的 Mod 有这个列表，对于知名 Mod 来说就算自己找也不麻烦，随便一搜就能找到。所以放弃这个想法吧，做这种功能意义不大...

**Comment by ruattd** [comment]
另外说一下，就算不确定这个新功能建议是否会被拒绝也请到 Issues 中提建议，善用分类会减轻社区的维护成本。

**Comment by z0z0r4** [comment]
无数据源，不可能批量地标记冲突 Mod，没人会给你做这个功能的
